styles/styles.css

```diff
@@ -0,0 +1,142 @@
+* {
+  font-family: 'Nanum Gothic', sans-serif;
+}
+
+li {
+  list-style-type: none
+}
+
+ul {
+  padding-inline-start: 0px;
+}
+
+h2 {
+  text-align: center
+}
+
+a:hover {
+  background-color: black
+}
+
+a:link {
+  color: grey;
+  text-decoration: none;
+}
+
+a:visited {
+  color: grey;
+  text-decoration: grey;
+}
+
+.navbar {
+  width: 100vw;
+  margin: 3rem;
+  display: grid;
+  grid-template: 1fr/ 1fr 1fr 1fr 1fr 1fr;
+  font-weight: bold;
+  text-align: left;
+} 
+
+
+#startly {
+  grid-column-start: 1;
+  grid-column-end: span 3;
+  text-align: left;
+}
+
+.title{
```

Comment:
By setting this up as a grid you avoided having to do weird things with the <sub>, that's great!